### DISCUSSION TITLE
markor-compatible editor for Linux

### DISCUSSION THREAD

**Discussion by kosakepferd** [gsantner/markor#2067] [Q&A] [answered]
Does anyone know a Linux text editor that is similar to Markor? I am thinking of synchronising my notes with my desktop (Linux Mint) using Syncthing. Therefore, it would be nice to have an editor that works similarly to Markor.

**Comment by gsantner** [comment]
Please search open discussions/issues. I recall multiple quite similar discussions with responses.

**Comment by ghost** [comment]
Emacs?

**Reply by kosakepferd** [reply]
This seems to me to be more suitable for programmers. I just want to capture my thoughts in text.

**Comment by TimReset** [comment]
I use JetBrains IDEA - a bit overkill, but it supports AsciiDoc (with plugin) and the rest of format that Markor supports, works in Linux, Windows, Mac. There is free opensource version. I have shared folder with my notes, and I open this folder as project in IDEA. It works perfectly!

**Comment by gsantner** [comment, marked as answer]
Marker (GTK3) is also a option.
